developer-resources/Spring.md

```diff
@@ -26,11 +26,11 @@ For example, this is the declaration used for the OMRS REST APIs
 @RequestMapping("/open-metadata/repository-services")
 ```
 
-This URI follows the root URL of the server.  So if the server was using `http://localhost:8080`, the methods are called
+This URI follows the root URL of the server.  So if the server was using `https://localhost:9443`, the methods are called
 using
 
 ```
-http://localhost:8080/open-metadata/repository-services ...
+https://localhost:9443/open-metadata/repository-services ...
```

Comment:
A big question here is whether we enable https ONLY or if we listen on BOTH ports. 

My initial thought was both - but given all interactions are moving to secured it's probably ok to do https only, as long as it 'just works' with self signed, works fine when developers debugging - in that case simply changing the ports is ok

Reply:
Interestingly if using a service mesh like istio in k8s there's a certain appeal in allowing individual containers to continue using simple http - without any need to deal with certificates & keystrokes, and using the mesh framework to manage the proxying through https. This is a pretty standard pattern. We may actually be making this a little more complicated by 'doing the right thing' and assuming https. It may not change the fact https is a good default, but I wonder if this is another reason we need to keep http - and figure out how to make configurable.

Will research some more on what is best practice